src/Microsoft.AspNet.Mvc.Core/InputFormatter.cs

```diff
@@ -18,6 +18,18 @@ namespace Microsoft.AspNet.Mvc
     public abstract class InputFormatter : IInputFormatter
     {
         /// <summary>
+        /// Returns UTF8 Encoding without BOM and throws on invalid bytes.
+        /// </summary>
+        public static readonly Encoding UTF8EncodingWithoutBOM
```

Comment:
I'd go for protected unless you have a strong reason to make them public. The point is to make it easy for subclasses to do the right thing right?